config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml

```diff
@@ -51,14 +51,14 @@ periodics:
       resources:
         requests:
           cpu: 6
-          memory: "39Gi"
+          memory: "16Gi"
```

Comment:
scale jobs have over-provisioning to guarantee they're not eviction candidates because evicting them costs us a ton of wasted external resources. we could do a priority class instead but this works fine for our purposes.

Reply:
I'll apply the standard 7 cpu and 30Gi to run on its own.

Also, we shouldn't be evicting pods if the scheduler can't find a free node but spin up new nodes.

Reply:
Preemption happened to a job I was checking yesterday. Scale jobs are some of the few where even rare preemption is realllly expensive both in wasted compute and in additional time to get signal.